geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/GeoMesaMetadata.scala

```diff
@@ -384,5 +384,5 @@ object MultiRowAccumuloMetadata extends AccumuloMetadataRow {
 
   override def getRange(typeName: String): Range = Range.prefix(s"$typeName~")
 
-  override def getTypeName(row: Text): String = Text.decode(row.getBytes, 0, row.find("~"))
+  override def getTypeName(row: Text): String = Text.decode(row.getBytes, 0, row.toString.lastIndexOf("~"))
```

Comment:
Since the field names are separate from the typename, I'm tempted to go ahead and merge this as is.

The typename which could go wrong is tower:type from OSM.